extensions/KHR/SPV_KHR_cooperative_matrix.asciidoc

```diff
@@ -227,9 +227,9 @@ are laid out with a fixed stride communicated via the 'Stride' operand to
 provided. Elements (row,$$*$$) of the result of load or store operations are taken
 in order from contiguous locations starting at 'Pointer'[row*'Stride'] where
 'Pointer' is the 'Pointer' operand to *OpCooperativeMatrixLoadKHR* or
-*OpCooperativeMatrixStoreKHR*. 'Stride' must be greater than 0 when passed to
-*OpCooperativeMatrixStoreKHR* and must be greater than or equal to 0 when passed
-to *OpCooperativeMatrixLoadKHR*.
+*OpCooperativeMatrixStoreKHR*. 'Stride' is interpreted as unsigned and must be
+greater than 0 when passed to
+*OpCooperativeMatrixStoreKHR*.
```

Comment:
We did discuss potentially requiring the stride be such that no write collisions occur. Do we want to do this too?